### ISSUE TITLE
command 'sfdx.force.project.create' not found

### ISSUE THREAD

**Issue by kbadamane121** [forcedotcom/salesforcedx-vscode#1670]
this issue has started after i installed the vs studio code and i am not able to create create any project using sfdx commands

**Comment by kbadamane121** [comment]
Thanks for your reply

   - Salesforce extensions for VSCode version:-46.15.1/previous was 46.14.0
   - VSCode version:-1.38.1
   - OS:-WIN 10

Cannot find en-us, defaulting to en

Activating extension 'salesforce.salesforcedx-vscode-apex' failed: Class
extends value undefined is not a constructor or null.
_logMessageInConsole @ abstractExtensionService.ts:391
console.ts:137 [Extension Host] Activating extension
`salesforce.salesforcedx-vscode-apex` failed:  Class extends value
undefined is not a constructor or null
Cannot activate extension 'Apex Interactive Debugger' because it depends on
extension 'salesforce.salesforcedx-vscode-apex', which failed to activate.
_logMessageInConsole @ abstractExtensionService.ts:391

abstractExtensionService.ts:391 Cannot activate extension 'Apex Replay
Debugger' because it depends on extension
'salesforce.salesforcedx-vscode-apex', which failed to activate.

**Comment by lcampos** [comment]
@kbadamane121 this is caused by your configuration enabling a setting called `salesforcedx-vscode-core.internal-development` which was meant to be used only by our team. I'll remove it in the next release but you should be able to get past this errors by removing it from your settings.json file.

**Comment by lcampos** [comment]
A fix to remove the setting and update the api so the Apex extension doesn't fail when that setting is in use has been released.

**Comment by pathamallas** [comment]
I m trying to create a project on palette its showing 'sfdx.force.project.create' not found

**Comment by alexiswilliams** [comment]
salesforcedx-vscode-core.internal-development is still showing up for me. I had to remove it as well from the settings JSON file.

**Comment by shivanand11totar** [comment]
![Untitled](https://user-images.githubusercontent.com/65186811/81826268-3c9e7d80-9555-11ea-8ef6-2d8381efd3d1.png)
![Untitled](https://user-images.githubusercontent.com/65186811/81826413-5fc92d00-9555-11ea-9ece-575197a882a2.png)

**Comment by shivanand11totar** [comment]
![Untitled](https://user-images.githubusercontent.com/65186811/81826510-7d969200-9555-11ea-91bb-f9074ecdd323.png)

**Comment by shivanand11totar** [comment]
![Untitled12](https://user-images.githubusercontent.com/65186811/81836126-aec88f80-9560-11ea-8f54-fd8648e0ddb3.png)

**Comment by AnanyaJha** [comment]
@shivanand11totar Thanks for reporting this! Are you still running into this problem? If so, can you tell us what version of the Salesforce CLI you're currently on? `sfdx --version`

**Comment by sfsholden** [comment]
Please also include any errors found in the `Output` for `Salesforce CLI`.

**Comment by shivanand11totar** [comment]
Thanks for the response i already solved it

**Comment by Tanmay1201** [comment]
@shivanand11totar I am facing the same issue, could you please tell that how did you resolved this?

**Comment by sfsholden** [comment]
@Tanmay1201 Please try updating to the latest version of the extensions - 48.17.0. Does this resolve the issue? 

If this does not solve the issue, please provide the following:
- Salesforce extensions for VSCode version
- VSCode version
- OS

Are there any errors showing in these places? If so, please paste them for us:
- Developer Tools from VSCode (Help > Toggle Developer Tools or Developer: Toggle Developer Tools in the command palette)
- Output Logs from VSCode (`Output: Focus on Output View` from the Command Palette). The logs we'd be interested in are `Log (Extension Host)` and `Salesforce CLI`.

**Comment by shivanand11totar** [comment]
installed nodejs and issue solved @Tanmay1201 @AnanyaJha

**Comment by rajvivora22** [comment]
I am facing similar issue 

Salesforce extensions for VSCode version: 49.1.0
VSCode version: 1.47.2
OS: Windows 10
I have installed jdk-11.0.8 and set up JAVA_HOME Path correctly in Environment variables.
Attaching snapshots of the errors. Please help!

![Errors](https://user-images.githubusercontent.com/68656978/88202526-a2e70100-cc66-11ea-954a-377dbc057fee.PNG)
![ProjectError](https://user-images.githubusercontent.com/68656978/88202593-be520c00-cc66-11ea-893b-efe9bde65adc.PNG)

**Comment by Droberts359** [comment]
I am also having this problem again. 

Operating system: MacOS
VS Code: 1.47.2
SF CLI: sfdx-cli/7.66.2-4f159a1d07 darwin-x64 node-v12.13.0


Dev Console errors: 
<img width="778" alt="Screen Shot 2020-07-23 at 7 39 44 PM" src="https://user-images.githubusercontent.com/29789357/88348773-4e05c200-cd1c-11ea-8ae6-67fd353ba9ca.png">

**Comment by sfsholden** [comment]
@rajvivora22 Thanks for reaching out and apologies for the delay. Please have a look at [Troubleshooting Common Issues](https://developer.salesforce.com/tools/vscode/en/troubleshooting).

It looks like we have a similar issue listed [here](https://github.com/forcedotcom/salesforcedx-vscode/issues/2379), which has been dupped to [this](https://github.com/forcedotcom/salesforcedx-vscode/issues/2295) issue.

**Comment by sfsholden** [comment]
@Droberts359 Thanks for reaching out and apologies for the delay.

Please also have a look at [Troubleshooting Common Issues](https://developer.salesforce.com/tools/vscode/en/troubleshooting) in case this is a problem with the CLI configuration. In the event that the troubleshooting suggestions and workarounds mentioned [here](https://github.com/forcedotcom/salesforcedx-vscode/issues/2295) do not unblock you, please open a new issue and we'd be happy to take a look.

**Comment by sfsholden** [comment]
This is a closed issue. Going forward please open new issues [here](https://github.com/forcedotcom/salesforcedx-vscode/issues). Our team is actively looking at new issues and are more than happy to help!

Thanks!

**Comment by vikki1221** [comment]
i am getting the error  sfdx.force.project.create not found can anyone help how to ressolve it
